docs/source/concept_models/vivarium_ciff_sam/concept_model.rst

```diff
@@ -149,13 +149,68 @@ Historically, prevention research has primarily focused on stunting, and, as a r
 ------------------------
 
 **Baseline**
-The baseline scenario will project GBD 2019 demographic and disease trends out from 2022 to 2027 and coverage rates for all preventive and therapeutic interventions will be held constant across the 5 years of the microsimulation to simulate a business-as-usual treatment scenario.
+The baseline scenario will project GBD 2019 demographic and disease trends and GBD 2020 exposure trends out from 2022 to 2027 and coverage rates for all preventive and therapeutic interventions will be held constant across the 5 years of the microsimulation to simulate a business-as-usual treatment scenario.
 
 **Alternative scenario 1**
-Scale up the coverage of MAM management and MAM treatment interventions to 90%
+Scale up the 'effective-coverage' of GAM treatment from baseline level of effective-coverage to scenario level of effective-coverage. Those who are 'effectively covered' have a shorter duration of SAM and MAM episodes. In this first alternative scenario, a larger proportion of simulants will be effectivey covered. Keeping incidence of MAM (i2) and SAM (i1) the same as baseline, we expect the prevalence of SAM and MAM to decrease  with a shorter duration (prevalence ~ incidence x duration).    
+
+| Effective coverage = treatment coverage x treatment efficacy
```

Comment:
What is the definition of treatment efficacy here? The proportion of treated individuals who respond to treatment? I think we should define this for clarity : )

Reply:
Got it, thanks!